test/jdk/com/sun/jdi/TestScaffold.java

```diff
@@ -461,44 +461,64 @@ protected void failure(String str) {
         testFailed = true;
     }
 
+    final List<String> doubleWordArgs = List.of("-cp", "-classpath", "--add-opens", "--class-path",
+            "--upgrade-module-path", "--add-modules", "-d", "--add-exports", "--patch-module", "--module-path");
+
     private ArgInfo parseArgs(String args[]) {
         ArgInfo argInfo = new ArgInfo();
-        String mainWrapper = System.getProperty("main.wrapper");
-        if ("Virtual".equals(mainWrapper)) {
-            argInfo.targetAppCommandLine = TestScaffold.class.getName() + " " + mainWrapper + " ";
-        } else if ("true".equals(System.getProperty("test.enable.preview"))) {
-            // the test specified @enablePreview.
-            argInfo.targetVMArgs += "--enable-preview ";
-        }
-
+        // Parse arguments, like java/j* tools command-line arguments
+        // the first argument not-starting with '-' is treated as a classname
+        // the other arguments are split to targetVMArgs targetAppCommandLine correspondingly
```

Comment:
Can you make these proper sentences with the first letter uppercase and a period at the end.